swagger.json

```diff
@@ -959,6 +959,182 @@
                 }
             }
         },
+        "/fleet/unassigned_driving_segments": {
```

Comment:
I talked quickly with @silverspace and I think we want to have this GET for the list, and then use `startMs` and `vehicleId` to create a single `PATCH` for a segment to change the annotation or pending driver assignment. I'll set up a time for us to quickly review.